packages/vaex-core/vaex/__init__.py

```diff
@@ -117,9 +117,10 @@ def open(path, convert=False, shuffle=False, copy_index=False, fs_options={}, *a
     :param str or list path: local or absolute path to file, or glob string, or list of paths
     :param convert: convert files to an hdf5 file for optimization, can also be a path
     :param bool shuffle: shuffle converted DataFrame or not
+    :param bool copy_index: copy index when source is read via pandas
```

Comment:
I don't think we want to have this here, this means it will be part of the API, and with arrow csv reading we will never have this again. we should only have arguments that are common (or very common) to all the dataset reading.

Reply:
That is not a new thing.. i just re-ordered it so that the order of the kwargs in the docstring matches the one in the function.

I suggest we remove it when `read_csv` is shifted from pandas to arrow. Also I don't know if we should fully drop `pandas.read_csv`, might depend on the type of options the arrow reader provides.

Reply:
Maybe we should prepare for this, read_cvs_pandas, read_cvs_arrow, and a read_csv with a backend='pandas' argument?

Reply:
Sounds good to me. I would propose `read_csv_pandas` and `read_csv_arrow` to be private method, and `read_csv` with a `backend` kwarg to be the one exposed to the world.